packages/swapper/src/swappers/ArbitrumBridgeSwapper/endpoints.ts

```diff
@@ -159,6 +158,9 @@ export const arbitrumBridgeApi: SwapperApi = {
     txHash,
     chainId,
     assertGetEvmChainAdapter,
+    getEthersV5Provider,
```

Comment:
why are we passing this function vs importing the singleton directly?

Reply:
Because swapper deps 😢

Reply:
easy to remove, but not necessary for this pr. threw me off since it was being added back from deps is all.

Reply:
9a73e2021fdd9b06045d067664dcc3b083fe5f1a